AirCasting/ABConnector/SDCardAirBeamServices.swift

```diff
@@ -0,0 +1,104 @@
+// Created by Lunar on 17/11/2021.
+//
+
+import CoreBluetooth
+import CoreMIDI
+
+enum SDCardSessionType: CaseIterable {
```

Comment:
let's move this out to a separate file or to a controller file (it's higher level, universal concept, doesn't really belong only to the Services part imo)